tests/unit_tests/udf_wrapper_params/token_classification/single_model_single_batch_complete.py

```diff
@@ -25,6 +25,27 @@ def udf_wrapper():
     def run(ctx: UDFContext):
         udf.run(ctx)
 
+def work_with_span_udf_wrapper():
```

Comment:
I think, we should write these tests directly using the Standalone Mock and not continue with these cursed tests